### DISCUSSION TITLE
Number speech consistently attributed to incorrect speaker

### DISCUSSION THREAD

**Discussion by moskeyombus** [deepgram/community#226] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

We recently started seeing an issue on some of our transcriptions where numerical values spoken by an individual on an audio channel are consistently attributed to an incorrect speaker.  This happens on files that are using diarization to identify speakers on the audio.  The behavior boils down to something like this:

```
Speaker A: The date is the 
Speaker B: 20th
Speaker A: of June
```
I've personally seen this on 2 different transcriptions, and the speech patterns of the individuals don't explain the behavior.  It's also clearly _only_ happening with numerical values.  It's worth noting, in the example linked below, I've been running this exact same audio file for testing over the past few months and did not encounter this issue until recently.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?callback=https%3A%2F%2Fus-central1-depodirect-dev.cloudfunctions.net%2Fscribe-transcript-receiver-function&diarize=true&model=general&punctuate=true&smart_format=true&tier=nova

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

4aa15331-b3f0-4e20-9e7a-9faa0e22a632

### If possible, please attach an example audio file to reproduce the issue.

mp3 not supported in this form, can send via email

**Comment by shirgoldbird** [comment]
Hi @moskeyombus, thanks for the bug report. Could you email me the mp3 file? My email is shir (dot) goldberg at deepgram (dot) com.

**Reply by moskeyombus** [reply]
@shirgoldbird file sent, thanks!

**Reply by shirgoldbird** [reply]
Thank you @moskeyombus! I've replicated this behavior and am passing it along to our engineering team to take a look.

**Reply by moskeyombus** [reply]
@shirgoldbird any update on this?  Seems to be happening to us pretty consistently and it's having an outsize impact on our product.  This issue was not present when we started using Deepgram, it showed up pretty recently, is there a feature or something on our account that got enabled that is causing this?  Perhaps we can disable something....

**Reply by shirgoldbird** [reply]
@moskeyombus It looks like this may be caused by an unanticipated interaction between our diarization model and our new smart formatting model. Our researchers are investigating some possible solutions, but I don't have a timeline for those at the moment. The quickest fix would be to disable smart formatting, if that's something that would be acceptable for your use case.

Sorry I don't have better news. I'll be sure to give you another update when I have it!

**Reply by moskeyombus** [reply]
@shirgoldbird Is there en estimated release date for the next round of updates for the Nova model?  Crossing my fingers and hoping there will be some resolution there, this is still an issue for us

**Reply by moskeyombus** [reply]
@shirgoldbird any update here?  This bug is still happening consistently, really need to know what's going on here

**Reply by shirgoldbird** [reply]
@moskeyombus Our teams are continuing to work on this. Unfortunately, I don't have an update on the release date for you at this time. I'll be sure to reach out as soon as I do.

**Comment by jpvajda** [comment, marked as answer]
@moskeyombus  With the release of our Nova-2 Model you might want to try that model for this problem you were facing.  Here's a [getting started guide for using Nova-2]( https://developers.deepgram.com/docs/nova-quickstart. )

If you still see an issue with Nova-2 please let us know.

**Reply by moskeyombus** [reply]
@jpvajda thanks.  This issue seems to have randomly disappeared months ago well before nova-2 was released 🤷
